bundles/org.openhab.binding.homeconnect/README.md

```diff
@@ -0,0 +1,310 @@
+
+![Screenshot Home Connect application page](doc/home_connect_application.png "Screenshot Home Connect application page")
+
+
+
+#### 3. Setup bridge (openHAB UI)
+
+The Home Connect bridge can be configured in the openHAB UI as follows:
+
+1. Go to the Inbox and press the add button
+2. Choose `Home Connect Binding`
+3. Select `Home Connect API`
+4. Setup and save thing
+    * __client id:__ your application client id
+    * __client secret:__ your application client secret
+    * __simulator:__ false
+5. Now navigate to the URL (`Redirct URI`) you've  added to your Home Connect application in the previous step (2.3). For example `http://192.168.178.80:8080/homeconnect`.
+6. Please follow the steps shown to authenticate your binding. You can redo this step every time. For example if you have authentication problems, just start wizard again.  
+![Screenshot Home Connect wizard page 1](doc/homeconnect_setup_1.png "Screenshot Home Connect wizard page 1")  
+![Screenshot Home Connect wizard page 2](doc/homeconnect_setup_2.png "Screenshot Home Connect wizard page 2")  
+![Screenshot Home Connect wizard page 3](doc/homeconnect_setup_3.png "Screenshot Home Connect wizard page 3")  
+![Screenshot Home Connect wizard page 4](doc/homeconnect_setup_4.png "Screenshot Home Connect wizard page 4")  
+
+7. That's it! Now you can use autodiscovery to add devices. Your devices should show up if you start a device scan in the openHAB UI.
+
+
+
+## Examples: File based configuration
+
+If you prefer to configure everything via file instead of openHAB UI, here are some examples.
+
+### things/homeconnect.things
+
+```
+Bridge homeconnect:api_bridge:api_bridge1 "Home Connect API" [ clientId="1234", clientSecret="1234", simulator=false] {
+    // Thing configurations
+    Thing dishwasher dishwasher1 "Dishwasher"  [ haId="SIEMENS-HCS02DWH1-6F2FC400C1EA4A" ]
+    Thing washer washer1 "Washer"  [ haId="SIEMENS-HCS03WCH1-1F35EC2BE34A0F" ]
+    Thing fridgefreezer fridge1 "Fridge Freezer  [ haId="SIEMENS-HCS05FRF1-7B3FA5EB3D885B" ]
+    Thing oven oven1 "Oven"  [ haId="BOSCH-HCS01OVN1-2132B6FA25BA21" ]
+    Thing dryer dryer1 "Dryer"  [ haId="BOSCH-HCS04DYR1-3921C766AD5BAF" ]
+    Thing coffeemaker coffee1 "Coffee machine"  [ haId="BOSCH-HCS06COM1-2140A8821AE7AB" ]
+    Thing washerdryer washerdryer1 "Washerdryer"  [ haId="BOSCH-HCS06COM1-2140A8821AE7AB" ]
+    Thing fridgefreezer fridgefreezer1 "Fridge/Freezer"  [ haId="BOSCH-HCS06COM1-2140A8821AE7AB" ]
+    Thing hood hood1 "Hood"  [ haId="BOSCH-HCS06COM1-2140A8821AE7AB" ]
+    Thing hob hob1 "Hob"  [ haId="BOSCH-HCS06COM1-2140A8821AE7AB" ]
+}
+```
+
+### items/homeconnect.items
+
+The channel parameter uses the following syntax: `homeconnect:<thing type id>:<bridge id>:<thing id>:<channel type id>`. For example: `homeconnect:dishwasher:simulator_api_bridge:dishwasher1:power_state`
```

Comment:
Just for consistency, please use `api_bridge1` as bridge id here and in your channel links.